easybuild/easyconfigs/m/MySQL/MySQL-5.7.21-GCCcore-6.4.0-clientonly.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'MySQL'
+version = '5.7.21'
+versionsuffix = '-clientonly'
+
+homepage = 'http://www.mysql.com/'
+description = """MySQL is one of the world's most widely used open-source
+ relational database management system (RDBMS)."""
+
+source_urls = ['https://downloads.mysql.com/archives/get/file/']
+sources = ['mysql-boost-%(version)s.tar.gz']
+patches = ['MySQL-5.7.21_Fix_ndb_dir_error.patch']
+checksums = [
+    'ad29ecb6fb3c3571394fe231633a2d1d188d49e9eb749daa4e8799b7630daa09',  # mysql-boost-5.7.21.tar.gz
+    '8e0ec125dd9cfa0a7ed845eb62b67add0bdd263d73929dff8eb12329ad7f59fb',  # MySQL-5.7.21_Fix_ndb_dir_error.patch
+]
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('CMake', '3.9.5'),
+]
+
+dependencies = [
+    ('libevent', '2.1.8'),
+    ('ncurses', '6.1'),
+    ('libedit', '20180525'),
+    ('zlib', '1.2.11'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# Boost headers are included in the source package and are only needed
+# at build time. No runtime dependencies
+# See http://mysqlserverteam.com/building-mysql-with-boost
+configopts = "-DWITHOUT_SERVER=ON -DWITH_EDITLINE=system \
+              -DWITH_BOOST=%(builddir)s/mysql-5.7.21/boost \
```

Comment:
@vanzod please use `%(version)s` rather than hardcoding the version

Reply:
@vanzod Also, we rarely use `\` in easyconfigs... Also, move the comment above the line it relates to:

```python
configopts = "-DWITHOUT_SERVER=ON -DWITH_EDITLINE=system -DWITH_ZLIB=$EBROOTZLIB "
# Boost headers are included in the source package and are only needed at build time (no runtime dependencies)
# see http://mysqlserverteam.com/building-mysql-with-boost
configopts += "-DWITH_BOOST=%(builddir)s/mysql-5.7.21/boost"